machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-azure.adoc

```diff
@@ -0,0 +1,67 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="cpmso-config-options-azure"]
+= Control plane configuration options for {azure-full}
+include::_attributes/common-attributes.adoc[]
+:context: cpmso-config-options-azure
+
+toc::[]
+
+You can change the configuration of your {azure-first} control plane machines and enable features by updating values in the control plane machine set.
+When you save an update to the control plane machine set, the Control Plane Machine Set Operator updates the control plane machines according to your configured xref:../../../machine_management/control_plane_machine_management/cpmso-managing-machines.adoc#cpmso-feat-config-update_cpmso-managing-machines[update strategy].
+
+[id="cpmso-sample-yaml-azure_{context}"]
+== Sample YAML for configuring {azure-full} clusters
+
+The following example YAML snippets show provider specification and failure domain configurations for an {azure-short} cluster.
+
+//Sample Azure provider specification
+include::modules/cpmso-yaml-provider-spec-azure.adoc[leveloffset=+2]
+
+//Sample Azure failure domain configuration
+include::modules/cpmso-yaml-failure-domain-azure.adoc[leveloffset=+2]
+
+[id="cpmso-supported-features-azure_{context}"]
+== Enabling {azure-full} features for control plane machines
+
+You can enable the following features by updating values in the control plane machine set.
```

Comment:
I can see the following content in the docs preview, but seems they are for AWS, no these fields for Azure.
```
providerSpec:
  value:
    loadBalancers:
    - name: lk4pj-ext 
      type: network 
    - name: lk4pj-int
      type: network
```

Reply:
Thanks! I started a [thread](https://redhat-internal.slack.com/archives/GE2HQ9QP4/p1713358148506019) in Slack about this to try to get it straightened out :)

Reply:
I have pushed a new commit to untangle the AWS and Azure spec differences, and add the right line for Azure. PTAL :bow: 
* [commit with the change](https://github.com/openshift/openshift-docs/pull/73646/commits/16da1f7aa96fe747c370d6bc0e025e7cdd068019)
* [Postinstallation version](https://73646--ocpdocs-pr.netlify.app/openshift-enterprise/latest/post_installation_configuration/configuring-private-cluster#private-clusters-setting-api-private_configuring-private-cluster) (both providers)
* [AWS CPMS version](https://73646--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-aws#private-clusters-setting-api-private_cpmso-config-options-aws)
* [Azure CPMS version](https://73646--ocpdocs-pr.netlify.app/openshift-enterprise/latest/machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-azure#private-clusters-setting-api-private_cpmso-config-options-azure)